universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -41,11 +42,10 @@
  * grouping those methods with the same type together.
  */
 public class TestQuestionBank {
-  private static final long VERSION = 1L;
+  private static final Map<QuestionEnum, Question> questionCache = new ConcurrentHashMap<>();
```

Comment:
Suggestion (blocking): I think this may be caused by merge conflict, we have moved this to be non static to resolve a bug where db is not used.

Reply:
Thanks, fixed!